sdk/communication/communication-sms/test/public/suites/smsClient.send.ts

```diff
@@ -19,7 +19,7 @@ export default function testCases(): void {
 
     assert.lengthOf(results, 1, "must return as many results as there were recipients");
     assertIsSuccessResult(results[0], validToNumber);
-  });
+  }).timeout(4000);
```

Comment:
identity v2 is slower so some tests that weren't timing out previously are now (in AAD)

Reply:
Thank you for pointing this out. I’m also aware of this. Today, your comment helped me realize there’s a possible performance improvement I could do. I’ve written the idea to my team! Thank you! (if it ends up happening, I’ll let you know)